### ISSUE TITLE
Alphabetize Recipes

### ISSUE THREAD

**Issue by Bartholomue** [StardewModders/mod-ideas#921]
While I love the idea of mods like Better Crafting, I'd like a mod that just alphabetizes the recipe list without changing or replacing the interface. I'm using an accessibility mod that can't handle screen alterations, so it consequently cannot read interfaces like Better Crafting. I've seen Tidy Crafting reorganize the recipe list from small to big craftable, so I hope that by using Content Code to inject C# code in to a content patch mod, this might be possible?

**Comment by holy-the-sea** [comment]
Hi! Is this still a mod that would be helpful for you?

**Comment by Bartholomue** [comment]
@holy-the-sea Hello. I would definitely still love this. I'm currently just using the tidy crafting mod which... doesn't exactly do what I'm looking for. Sorry for the ages late delay. I stopped playing Stardew for a while and hadn't been paying attention to Github.
